src/app/home-page/home-logged-out/home.component.scss

```diff
@@ -111,6 +111,11 @@ h3 {
   border-radius: 1rem;
 }
 
+.close-alignment {
```

Comment:
try to put the button without the close-alignment classes in any component. https://stackblitz.com/angular/nlkqgebarxk?file=src%2Fapp%2Finput-clearable-example.ts has no styling and looks close to what it should be.

Reply:
This is how it looks like when I don't do any additional styling for the button
<img width="583" alt="Screen Shot 2021-05-28 at 4 52 15 PM" src="https://user-images.githubusercontent.com/65895867/120040721-11a69080-bfd5-11eb-8b4d-babdcccce3eb.png">

Reply:
figure out what styling is messing with it and remove that style instead of adding more styles

Reply:
okay, thank you. will try to figure it out